core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -736,8 +743,8 @@ object TopicCommand extends Logging {
         allTopicLevelOpts -- Set(describeOpt) ++ allReplicationReportOpts - reportUnavailablePartitionsOpt + topicsWithOverridesOpt)
       CommandLineUtils.checkInvalidArgs(parser, options, topicsWithOverridesOpt,
         allTopicLevelOpts -- Set(describeOpt) ++ allReplicationReportOpts)
-      CommandLineUtils.checkInvalidArgs(parser, options, ifExistsOpt, allTopicLevelOpts -- Set(alterOpt, deleteOpt, describeOpt) ++ Set(bootstrapServerOpt))
-      CommandLineUtils.checkInvalidArgs(parser, options, ifNotExistsOpt, allTopicLevelOpts -- Set(createOpt) ++ Set(bootstrapServerOpt))
```

Comment:
Would it be easy to add a for this? (ensure the bootstrap server doesn't throw)

Reply:
Sorry... what do you mean by `add a for this`?

Reply:
I think a word got left out here :)

Reply:
I think you meant a test.. if so, the tests I added already use --bootstrap-server with these options..